ts-obs/tests/main_test.go

```diff
@@ -6,8 +6,12 @@ import (
 	"os/exec"
 	"os/signal"
 	"testing"
+	"time"
 )
 
+const RELEASE_NAME = "gg"
+const NAMESPACE = "ns"
```

Comment:
do we have tests that do concurrent installs (with diff release names) or it will require too much resources from the minikube vm?

Reply:
I think its possible, but would require a good amount of refactoring. Maybe for the next pr?